cmd/justification/main.go

```diff
@@ -64,12 +65,17 @@ func realMain(ctx context.Context) error {
 		return fmt.Errorf("failed to setup kms client: %w", err)
 	}
 
+	fsClient, err := firestore.NewClient(ctx, cfg.ProjectID)
```

Comment:
I think we need to close these clients when the server shuts down. Applies to KMS too.